app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def comments_count
+    post.increment!(:Commets_counter)
+  end
```

Comment:
- [ ] there is a small typo in line 6 it should be `:Comments_counter` instead of `Commets_counter`